lib/cdo/hamburger.rb

```diff
@@ -296,4 +296,18 @@ def self.get_header_contents(options)
       signed_out_links
     end
   end
+
+  def self.get_farsi_header_contents(options)
+    loc_prefix = options[:loc_prefix]
+
+    signed_out_links = [
+      {title: I18n.t("#{loc_prefix}teach"), url: CDO.code_org_url("/global/fa/teacher"), id: "header-teach"},
+      {title: I18n.t("#{loc_prefix}about"), url: CDO.code_org_url("/global/fa/about"), id: "header-about"},
+      {title: "CSF", url: CDO.code_org_url("/global/fa/csf"), id: "header-csf"},
```

Comment:
@kelbyhawn how do I translate these? I think I remember the header being special